uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/app.js

```diff
@@ -0,0 +1,28 @@
+define([
+  'common/angular',
+  'common/angularBootstrap',
+  'common/services/angular-date/date-format',
+  'leave-absences/shared/ui-router',
+  'leave-absences/absence-tab/modules/settings',
```

Comment:
For other apps config contains the routing, which we dont need for absence tab. Thats why removed it. If it is required in future we can create.